### PR TITLE
Close # 42 Updated key variables and included Centiloid for all SUVR results

### PR DESCRIPTION
**Pull request by jwang-lilly**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: inst/templates/ad_adapet.R [hunk 1]
```diff
@@ -98,7 +100,7 @@ adapet <- adapet %>%
 
 ## Amyloid-specific derivations ----
 ### Convert SUVR to Centiloid ----
-adapet <- adapet %>%
+adapet_03 <- adapet_02 %>%
```
Comment (Fanny-Gautier) [inline inst/templates/ad_adapet.R:103]: Why is `"SUVRBFTP"` not converted here ? There are 5 different `SUVR` parameters in the `lookup_param` table, and only 4 Centiloid derived.

File: inst/templates/ad_adtpet.R [hunk 2]
```diff
@@ -41,11 +41,15 @@ nv <- metatools::combine_supp(nv, suppnv)
...
+  "SUVR", "FBB", "NEOCORTICAL COMPOSITE", "Whole Cerebellum", "AVID FBB SUVR PIPELINE", "SUVRAFBB", "AVID FBB Standard Uptake Ratio Neocortical Composite Whole Cerebellum", 2,
+  "SUVR", "FBP", "NEOCORTICAL COMPOSITE", "Whole Cerebellum", "BERKELEY FBP SUVR PIPELINE", "SUVRBFBP", "BERKELEY FBP Standard Uptake Ratio Neocortical Composite Whole Cerebellum", 3,
+  "SUVR", "FBB", "NEOCORTICAL COMPOSITE", "Whole Cerebellum", "BERKELEY FBB SUVR PIPELINE", "SUVRBFBB", "BERKELEY FBB Standard Uptake Ratio Neocortical Composite Whole Cerebellum", 4,
+  "SUVR", "FTP", "NEOCORTICAL COMPOSITE", "Inferior Cerebellar Gray Matter", "BERKELEY FTP SUVR PIPELINE", "SUVRBFTP", "BERKELEY FTP Standard Uptake Ratio Neocortical Composite Inferior Cerebellar Gray Matter", 5,
+  "VR", "FBP", NA, NA, "FBP VISUAL CLASSIFICATION", "VRFBP", "FBP Qualitative Visual Classification", 6,
+  "VR", "FBB", NA, NA, "FBB VISUAL CLASSIFICATION", "VRFBB", "FBB Qualitative Visual Classification", 7,
+  "VR", "FTP", NA, NA, "FTP VISUAL CLASSIFICATION", "VRFTP", "FTP Qualitative Visual Classification", 8 # FTP Visual was dropped in issue 18_nv_improve
```
Comment (Fanny-Gautier) [inline inst/templates/ad_adtpet.R:51]: Please remove unnecessary comments and include them in the conversation instead

**Review by Fanny-Gautier — CHANGES_REQUESTED** [review]
Derivations to be double checked as per available data

**Comment by Fanny-Gautier** [comment]
@jwang-lilly I echo Lina, I am also confused now. Please see my [comment](https://github.com/pharmaverse/admiralneuro/issues/42#issuecomment-3136123506) in the corresponding issue.

**Comment by Lina2689** [comment]
> @Fanny-Gautier, @Lina2689, It seems there has been some confusion regarding SUVR values in nv_neuro and the Centiloid computation in inst/templates/ad_adapet.R, I hope the following explanation help clarify the situation.
> 
> As of the main branch, we have the following combinations of tracers (FBB, FBP and FTP), pipelines (AVID and BERKELEY), and modalities - either VR (visual quantitation, positive or negative) or SUVR (numerical values) in nv_neuro:
> 
> {D6C126E7-3F6A-45B0-BFF5-9C7FF39BF4D6}
> Please note the following:
> 
> * FBB/AVID, and FBP/BERKELY were introduced in issue_18_nv_impove
> * FTP VR was dropped in issue_18_nv_improve
> * Original issue 7, I didn't include FBB/AVID in nv_neuro because at Lilly, we primarily use FBP, rarely use FBB, and I was not familiar with the SUVR to Centiloid conversion for FBB using the AVID pipeline. As a result, there was no corresponding FBB AVID SUVR to Centiloid equation in compute_centiloid().
> * Once FBB/AVID was added in issue_18_nv_improve, issue 45 was created to update compute_centiloid().
> * While working on ad_adapet.R, I discovered a typo in the NVMETHOD column ("VISIUAL" instead of VISUAL) in nv_neuro, which led to mismatches between nv_neuro and lookup table and dropped rows during the derive_vars_merged_lookup() step in inst/templates/ad_adapet.R. Issue 49 was created to fix the typo to address it.
> 
> However, there is still one row in the lookup table (line 55 in ad_adapet.R, corresponding to VR and FTP) that has no corresponding records in nv_neuro due to changed made in issue_18_nv_improve. Ideally, these records should be restored in a future update. These records are generally expected to align with FBP and FTP SUVR, specifically, the FTP VR positive records should not contradict their corresponding SUVR values.
> 
> If your branch doesn't include merged changes from issues 18, 45 and 49 PRs, that may explain the discrepancies you're seeing. I think syncing your branch with main should resolve some confusion.
> 
> Please let me if above rational makes sense to you.

Thanks @jwang-lilly. I sync the branch with main and its good now.

**Comment by jwang-lilly** [comment]
@Lina2689 @Fanny-Gautier, Are you happy with issue 42 pull request? Please approve it if no further changes are needed.

**Review by Fanny-Gautier — COMMENTED** [review]
OK with me if we don’t need to keep `AGTRT` and `AGCAT` populated for converted parameters.
If it makes sense to keep them populated, please go ahead and implement it.
Thanks.

File: inst/templates/ad_adapet.R [hunk 3]
```diff
@@ -126,25 +127,40 @@ adapet <- adapet %>%
...
-    #         ref_region = "Whole Cerebellum",
-    #         suvr = AVAL
-    #       ),
-    #       PARAMCD = "CLBFBP",
-    #       PARAM = "Centiloid (CL) based on BERKELEY FBP",
-    #       AVALU = "CL"
-    #     )
-    #   )
-    # ),
+    derivation_slice(
```
[the remark this answers is not in the document]
Comment (Fanny-Gautier) [reply inst/templates/ad_adapet.R:130]: ok, Thanks for the clarification. All good for me then!
